CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Features
```

Comment:
I'd say its a fix, users can't create it right now.

Reply:
Changed